source/core/security-internal-authentication.txt

```diff
@@ -18,6 +18,11 @@ You can authenticate members of :term:`replica sets <replica set>` and
 the members, MongoDB can use either keyfiles or :ref:`x.509 <security-auth-x509>`
 certificates.
 
+The selected method is used for all internal communication. For example, when a
+client authenticates to a :binary:`~bin.mongos`, the ``mongos`` then uses the
```

Comment:
After "a client authenticates to a :binary:`~bin.mongos`", probably add "using the supported client Authentication Mechanisms" and link Authentication Mechanisms to https://docs.mongodb.com/manual/core/authentication/#authentication-mechanisms ?